packages/mgt/src/graph/graph.people.ts

```diff
@@ -79,13 +149,24 @@ export async function findPeople(
  */
 export async function getPeople(graph: IGraph): Promise<Person[]> {
   const scopes = 'people.read';
+  // debugger;
+  const cache = CacheService.getCache<CachePeopleQuery>(cacheSchema, 'peopleQuery');
+  // not a great way to do this, don't know a better way
+  const cacheRes = peopleCacheEnabled ? await cache.getValue('*') : null;
+
+  if (cacheRes && peopleInvalidationTime > Date.now() - cacheRes.timeCached) {
+    return cacheRes.results.map(ppl => JSON.parse(ppl));
+  }
```

Comment:
Consider wrapping this entire block with `if (cacheEnabled)...`. 

Same for other functions